src/aselo-webchat.tsx

```diff
@@ -179,4 +183,6 @@ export const initWebchat = async () => {
 
   // Render WebChat
   webchat.init();
+
+  FlexWebChat.MessageList.Content.add(<Test key="test" />);
```

Comment:
Can we wrap this with a flag so that this change is not visible if deployed? You can have it hardcoded like:

```
const enable_exit_buttons = false;
if (enable_exit_buttons) {
  FlexWebChat.MessageList.Content.add(<Test key="test" />);
}
```